stable/kubernetes-dashboard/templates/ingress.yaml

```diff
@@ -25,6 +26,14 @@ spec:
               serviceName: {{ $fullName }}
               servicePort: {{ $httpPort }}
   {{- end -}}
+  {{- else }}
+  - http:
```

Comment:
The indentation of `http` isn't the same as `host`. I understand that they are in an if/else statement so they won't be present at the same time. But, the `path` below and the `host` above use the same indentation. This one should as well. Can you please indent this section 2 more spaces.

Reply:
Thank you for review! I have indented 2 spaces more.